frontend/device/kobo/device.lua

```diff
@@ -152,6 +152,8 @@ local Kobo = Generic:extend{
     hasEclipseWfm = no,
     -- Device ships with various hardware revisions under the same device code, requiring automatic hardware detection...
     automagic_sysfs = false,
+    -- Device is a Clara or Libra Colour
```

Comment:
Ehhh, I don't care for that comment. I'm not sure if one is necessary but if so it should probably just talk of RBG vs BGR.